### ISSUE TITLE
Please upgrade to .NET Core 3.preview8

### ISSUE THREAD

**Issue by donhuvy** [AnkitSharma-007/ASPCore.BlazorCrud#2]
Dear author,

![image](https://user-images.githubusercontent.com/1328316/64061541-6ea9c280-cc06-11e9-8863-092a149fc22a.png)

I try learning by using current stack: Visual Studio 2019 preview, dotNET Core 3 preview8, but error. Many sections in tutorial at here https://ankitsharmablogs.com/google-authentication-and-authorization-in-server-side-blazor-app/ is out of date and very hard to follow for new commer.

Please upgrade tutorial use new tools, new .NET Core version.

Thank you very much!

**Comment by AnkitSharma-007** [comment]
The repo has been updated to ```.NET Core 3.2 Preview-1```. Let me know if you face any issues.
